install_script.sh.template

```diff
@@ -770,7 +772,9 @@ if [ "$OS" == "RedHat" ]; then
 
     echo -e "  \033[33mInstalling package(s): ${packages[*]}\n\033[0m"
 
-    $sudo_cmd yum -y --disablerepo='*' --enablerepo='datadog' install $dnf_flag "${packages[@]}" || $sudo_cmd yum -y install $dnf_flag "${packages[@]}"
+    $sudo_cmd yum -y --disablerepo='*' --enablerepo='datadog' install $dnf_flag "${packages[@]}" 2>"/tmp/ddog_install_error_msg" || $sudo_cmd yum -y install $dnf_flag "${packages[@]}" 2>"/tmp/ddog_install_error_msg"
+    cat "/tmp/ddog_install_error_msg"
```

Comment:
Shouldn't we remove these `cat`. [I was told we prefer dogs (sorry for this one)]
The only display is the one in the final error message, correct?